src/components/challenges/active.tsx

```diff
@@ -27,26 +29,46 @@ type ActiveProps = {
     }[];
     is_active: boolean;
     is_user_subscribed: number;
-  };
+  },
+  userId: string
 };
 
-const Active: FC<ActiveProps> = ({ content }) => {
+const { SUCCESS, ERROR } = ToastTypes;
+
+const Active: FC<ActiveProps> = ({ content, userId }) => {
   const [isUserSubscribed, setUserSubscribed] = useState(content.is_user_subscribed);
 
+  const subscibeUser = async () => {
+    const data = {
+      challenge_id: content.id,
+      user_id: userId,
+    };
+    try {
+      const response = await fetch(SUBSCRIBE_TO_CHALLENGE_URL, {
```

Comment:
We have our own implementation of the `fetch` [here](https://github.com/Real-Dev-Squad/website-status/blob/develop/src/helperFunctions/fetch.ts), please use the same.